test/libsolidity/syntaxTests/tupleAssignments/tuple_in_tuple_long.sol

```diff
@@ -1,12 +1,11 @@
 contract C {
-  function f() {
+  function f() public {
     (((((((((((,2),)),)),),))=4)));
   }
 }
 // ----
-// SyntaxError: (15-69): No visibility specified. Did you intend to add "public"?
-// TypeError: (46-47): Expression has to be an lvalue.
```

Comment:
I'm a bit concerned that this error here is gone now.
On the other hand, this was just a test against a failing assertion anyway.

Reply:
Ah sorry, the error is still there.